script/worker

```diff
@@ -62,14 +62,13 @@ print help
 
 =head1 DESCRIPTION
 
-lorem ipsum ...
+(no content)
 
 =head1 CONFIG FILE
 
-L<OpenQA::Client> tries to find a config file in
-$OPENQA_CLIENT_CONFIG, ~/.config/openqa/client.conf or
-/etc/openqa/client.conf and reads whatever comes first.
-You can put API key and secret in that config file.
+Like L<OpenQA::Client> L<worker> uses a config file workers.ini resolving
+$OPENQA_CONFIG or a default /etc/openqa/. You can put API key and secret in
+that config file.
```

Comment:
Workers actually uses both files. For workers.ini it only looks into `$OPENQA_CONFIG` or `/etc/openqa`, for client.conf it uses `OpenQA::Client` to find so `~/.config/openqa/client.conf` is also doable.

I don't think you can put API key and secret into workers.ini.